system/commands/users.go

```diff
@@ -165,6 +165,10 @@ func Users(ctx context.Context, app serviceInitializer) *cobra.Command {
 					cli.HandleError(err)
 				}
 			}
+
+			if flagMakePasswordLink {
+				authSvc.SendPasswordResetToken(ctx, user.Email)
```

Comment:
I see what you wanted to do... But...

`--make-password-link` provides link to create password. It does NOT send any emails.
And you are abusing password-reset functionality and email for add-user action.

If we really want to do this then
 - create a new email template for this purpuse
 - and create another flag, like `--send-password-link`.